test/validate_config_test.js

```diff
@@ -13,7 +13,7 @@ describe("validateConfig", function() {
 
   it("returns false if no files exist", function(done) {
     temp.mkdir("no-config", function(err, directory) {
-      if (err) throw err;
+      if (err) {throw err;}
```

Comment:
Thanks for addressing these lint issues, can you add some spaces so it's `{ throw err; }`?